spec/Store.spec.js

```diff
@@ -532,21 +532,34 @@ describe('Store', () => {
         }
       })
 
-      it('adds server errors to the right record', async () => {
```

Comment:
we still want this test - looks like you accidentally overwrote it

Reply:
whoops! 🙈